galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -234,7 +234,7 @@ public void runTest(Properties bootstrapProperties, Properties overridePropertie
             testClass = getTestClass(testBundleName, testClassName);
         } catch(Throwable t) {
             logger.error("Problem locating test " + testBundleName + "/" + testClassName, t);
-            updateStatus("finished", "finished");
+            updateStatus(TestRunLifecycleStatus.FINISHED, "finished");
```

Comment:
thought we were going to eliminate this second string parameter ?

Reply:
part of another issue